chunks/chunks.go

```diff
@@ -309,11 +312,11 @@ func newReader(bs []ByteSlice, cs []io.Closer, pool chunkenc.Pool) (*Reader, err
 }
 
 // NewReader returns a new chunk reader against the given byte slices.
-func NewReader(bs []ByteSlice, pool chunkenc.Pool) (*Reader, error) {
+func NewReader(bs []ByteSlice, pool chunkenc.Pool, size int64) (*Reader, error) {
 	if pool == nil {
 		pool = chunkenc.NewPool()
 	}
-	return newReader(bs, nil, pool)
+	return newReader(bs, nil, pool, size)
```

Comment:
why do you need to take the size an argument to `func NewReader(bs []ByteSlice, pool chunkenc.Pool, size int64)` ?

why not just pass 0  to `return newReader(bs, nil, pool, 0)`